suites/test_module_content.py

```diff
@@ -126,8 +127,7 @@ def verify_module_content(self, api_auth, setup_test_products):
                      contains_string(assembly_title_prefix) or contains_string(assembly_prefix))
           check_that("isPartOf data", data_from_published_module.json()["module"]["isPartOf"][i],
                      all_of(has_entry("title"), has_entry("uuid"), has_entry("url"), has_entry("view_uri"),
-                            has_entry("relative_url"), has_entry("pantheon_env")))
-          check_that("isPartOf-> relative_url", published_assembly_url,
-                     contains_string(data_from_published_module.json()["module"]["isPartOf"][i]["relative_url"]))
-          check_that("isPartOf-> relative_url",
```

Comment:
@noopurmath3 this might fail too if we have one assembly included in 2 guides which is not the case at this moment. but it's the same logic as above. Please modify accordingly. thanks.

Reply:
updated the logic for verifying relative_url